types/vnode.d.ts

```diff
@@ -3,9 +3,7 @@ import { Vue } from "./vue";
 export type ScopedSlot = (props: any) => VNodeChildrenArrayContents | string;
 
 export type VNodeChildren = VNodeChildrenArrayContents | [ScopedSlot] | string;
```

Comment:
I wonder whre is `[ScopedSlot]` used. VNodeChildren is used in createElement function. But it seems scopedSlots should be defined in another argument rather than VNodeChildren. https://vuejs.org/v2/guide/render-function.html#createElement-Arguments

Reply:
Ok I found it 
https://github.com/vuejs/vue/blob/dev/src/core/vdom/create-element.js#L81-L83